tgui/packages/tgui/styles/interfaces/SecureStorage.scss

```diff
@@ -0,0 +1,68 @@
+@use '../base.scss';
+@use '../colors.scss';
+@use '../functions.scss' as *;
+
+$color-danger: colors.bg(colors.$red) !default;
+$color-caution: colors.bg(colors.$yellow) !default;
+
+$background-beige: #e8e4c9;
+
+.SecureStorage__displayBox {
+  background-color: #212121;
+  color: #8b8b8b;
+  border: 0.167em inset $background-beige;
```

Comment:
I hate a lot of the magic numbers in this file, it makes things not look right if shifted around, but its annoying to get them all to flex properly. I'll need to look more into alternate solutions

Reply:
So for this class, you are using a percentage for font-size that translates to 60px. With base 60, your 0.167em should translate close to about 10px. Maybe this would work better? This is just what I gleamed from looking at em to px conversions.